src/satosa/frontends/openid_connect.py

```diff
@@ -165,6 +164,9 @@ def register_endpoints(self, backend_names):
         :rtype: list[(str, ((satosa.context.Context, Any) -> satosa.response.Response, Any))]
         :raise ValueError: if more than one backend is configured
         """
+        provider_config = ("^.well-known/openid-configuration$", self.provider_config)
+        jwks_uri = ("^{}/jwks$".format(self.endpoint_basepath), self.jwks)
```

Comment:
So there will be no difference between the external (authorization, ...) and internal (token, jwks, ...) urls.

If someone runs satosa as local uwsgi service at `http://127.0.0.1:3030` and serve the external requests at `1.2.3.4` via a reverse proxy (nginx) under the `/satosa` location then the external requests has to call `http://1.2.3.4/satosata/.../authorization` and the internal calls has to use the same `/satosa` prefix: `http://127.0.0.1:3030/satosa/.../token`.

Reply:
However it may be cleaner to have to same routing rules for all the endpoints.